client/src/js/directives/bhSubmit.js

```diff
@@ -42,9 +42,9 @@ function bhSubmitDirective() {
         // the response is a promise, toggle the loading state on
         // fulfillment/rejection
         if (isPromise(response)) {
-          response.finally(function () {
-            FormController.$toggleLoading();
-          });
+          response
```

Comment:
Because the error is not being catched anywhere if I try to reject the promise on the unit test I get the following error:

![Screen Shot 2021-01-24 at 08 50 59](https://user-images.githubusercontent.com/4948776/105637972-fb35d000-5e4e-11eb-8375-fd587757d242.png)